tests/BenchmarkDotNet.IntegrationTests/BenchmarkSwitcherTest.cs

```diff
@@ -124,7 +125,59 @@ public void WhenUserAsksToPrintAListAndProvidesAFilterWePrintFilteredList()
             Assert.Contains("BenchmarkDotNet.IntegrationTests.ClassA.Method1", logger.GetLog());
             Assert.DoesNotContain("BenchmarkDotNet.IntegrationTests.ClassA.Method2", logger.GetLog());
         }
-        
+
+
+        [Fact]
+        public void WhenDisableLogFileWeDontWriteToFile()
+        {
+            var logger = new OutputLogger(Output);
+            var config = ManualConfig.CreateEmpty().With(logger).With(ConfigOptions.DisableLogFile);
+            string logFilePath = null;
+            try
+            {
+                var summaries = BenchmarkSwitcher
+                    .FromTypes(new[] { typeof(ClassA) })
+                    .RunAll(config);
+
+                var summary = summaries.Single();
+                logFilePath = summary.LogFilePath;
+                Assert.False(File.Exists(logFilePath), $"Logfile '{logFilePath}' should not exist, but it does.");
+            }
+            finally
+            {
+                if (!string.IsNullOrWhiteSpace(logFilePath))
+                {
+                    File.Delete(logFilePath);
```

Comment:
I would move this logic before the try block, to ensure that the file does not exist before calling `RunAll`

Reply:
I was thinking about that issue as well, but unfortunately the file name pattern generation isn't accessible. Thus, generating the file name outside within the test context would break the test, even if the assertion of it would still be valid.

Maybe I missed the piece of how I would do that or my thoughts are fundamentally wrong - could you please elaborate on this?

Reply:
Good point, I missed that. I think that we can stay with the current implementation.